scripts/CEE/get-kafka-instance-state/get-kafka-instance-state.sh

```diff
@@ -42,6 +42,22 @@ function main() {
   oc exec -n "${resource_namespace}" "${kafkaStatefulSet}" -c kafka -- sh /opt/kafka/bin/kafka-acls.sh \
     --bootstrap-server localhost:9096 --list > "${inspectDir}"/kafka-acls.txt 2>&1
 
+  oc exec -n "${resource_namespace}" "${kafkaStatefulSet}" -c kafka -- sh /opt/kafka/bin/kafka-transactions.sh \
+    --bootstrap-server localhost:9096 list > "${inspectDir}"/kafka-transactions-list.txt 2>&1
```

Comment:
Would it be worth gathering all transaction related files into a specific sub directory within `inspectDir`

Reply:
Maybe :thinking: I'll consider if that's worth it after I (maybe) move the `describe` stuff out of here to a separate script.

Reply:
Done, thanks for the suggestion!